src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -127,13 +132,19 @@ const AssigneeSelector = createReactClass({
     });
   },
 
-  assignTo(member) {
-    assignTo({id: this.props.id, member});
+  assignToUser(user) {
+    assignToUser({id: this.props.id, user});
+    this.setState({filter: '', loading: true});
+  },
+
+  assignToTeam(team) {
+    assignToActor({actor: {id: team.id, type: 'team'}, id: this.props.id});
     this.setState({filter: '', loading: true});
   },
 
   clearAssignTo() {
-    assignTo({id: this.props.id});
+    //clears assignment
+    assignToUser({id: this.props.id});
```

Comment:
Can you write a separate function? Should be too much boilerplate, and would read much more clearly